wp-content/plugins/core/src/Blocks/Types/Support/Media_Text_Media_Embed.php

```diff
@@ -6,12 +6,11 @@
 use Tribe\Gutenpanels\Blocks\Block_Type_Interface;
 use Tribe\Gutenpanels\Blocks\Sections\Content_Section;
 use Tribe\Project\Blocks\Block_Type_Config;
+use Tribe\Project\Blocks\Types\Media_Text;
 
 class Media_Text_Media_Embed extends Block_Type_Config {
 	public const NAME = 'tribe/media-text--media-embed';
 
-	public const EMBED = 'embed';
-
 	public function build(): Block_Type_Interface {
 		return $this->factory->block( self::NAME )
 			->set_label( 'Video' )
```

Comment:
Translatable?

Reply:
@jbrinley, I'm not really familiar with this Block code. Should this string and the one in Sam's next comment be translated?

Reply:
Yup, they should be translated. Apologies for that.